### DISCUSSION TITLE
Please refund my last payment for the copilot

### DISCUSSION THREAD

**Discussion by yura** [community/community#53810] [Copilot Conversations]
### Select Topic Area

General

### Body

I was unable to try it due to

```
GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"
```

TIA

**Comment by Akash1134** [comment]
Hi @yura ,

You can [Cancel your GitHub Copilot subscription](https://docs.github.com/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription#canceling-your-github-copilot-subscription) at any time.
- If you cancel during your 60 day trial, you won't be charged.
- If you do not cancel your GitHub Copilot subscription before the end of the 60 day trial, you will be charged according to the billing cycle you selected when setting up the trial.

For more information, refer to the [About billing for GitHub Copilot](https://docs.github.com/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot) documentation.

If you require further assistance, please [contact our billing team.](https://support.github.com/contact/account) We will try to process your requests as soon as possible but there may be a delay of several days. We really appreciate your patience during this busy period.
